contracts/snapshots/MerkleSnapshot.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.11;
+
+import "../zeppelin//MerkleProof.sol";
+import "../ManagerProxyTarget.sol";
+
+
+contract MerkleSnapshot is ManagerProxyTarget{
+    mapping (bytes32 => bytes32) public snapshot;
+
+    constructor(address _controller) public Manager(_controller) {}
+
+    function setSnapshot(bytes32 _id, bytes32 _root) external onlyControllerOwner {
+        require(snapshot[_id] == bytes32(0), "Snapshot root for ID already exists");
+        snapshot[_id] = _root;
+    }
+
+    function verify(bytes32 _id, bytes32[] calldata _proof, bytes32 _leaf) external view returns (bool) {
+        require(
```

Comment:
Since this is defined as a view function that returns a boolean could we just let the caller deal with handling the boolean return value? 

i.e.

```solidity
function verify(bytes32 _id, bytes32[] calldata _proof, bytes32 _leaf) external view returns (bool) {
    return MerkleProof.verify(_proof, snapshot[_id], _leaf);
}
```

I think an additional benefit of doing this is that if when this function is called off-chain the caller just has to handle the returned boolean (I'm not sure what the current behavior is when the function reverts for an `eth_call` - I recall there being some funky behavior in the past).

Reply:
fixed in 2839cec